src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbolBase.cs

```diff
@@ -142,28 +165,49 @@ private enum Flags : byte
                 _name = _lazySourceName = memberName;
             }
 
-            if ((isAutoProperty && hasGetAccessor) || hasInitializer)
+            if (isAutoProperty || hasInitializer)
             {
                 Debug.Assert(!IsIndexer);
-                string fieldName = GeneratedNames.MakeBackingFieldName(_name);
-                BackingField = new SynthesizedBackingFieldSymbol(this,
-                                                                      fieldName,
-                                                                      isReadOnly: (hasGetAccessor && !hasSetAccessor) || isInitOnly,
-                                                                      this.IsStatic,
-                                                                      hasInitializer);
+                _ = CreateBackingField(isCreatedForFieldKeyword: false);
             }
 
             if (hasGetAccessor)
             {
-                _getMethod = CreateGetAccessorSymbol(isAutoPropertyAccessor: isAutoProperty, diagnostics);
+                _getMethod = CreateGetAccessorSymbol(isAutoPropertyAccessor: isAutoProperty, diagnostics); // PROTOTYPE: Getter and setters may have different values of isAutoPropertyAccessor.
```

Comment:
>// PROTOTYPE: Getter and setters may have different values of isAutoPropertyAccessor. [](http://example.com/codeflow?start=107&length=85)

It all depends what we would like the ```isAutoPropertyAccessor``` parameter to mean. The meaning of the parameter shouldn't just change automatically because the spec changed the meaning of what "auto-property" means.

Reply:
After some digging around, it looks to me that we would like this parameter to have the following meaning - this is a semicolon only accessor for which compiler is supposed to generate its body. I.e. the following condition should be true ```!(containingType.IsInterface && !IsStatic) && !IsAbstract && !IsExtern && !isIndexer```